DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -137,6 +137,20 @@ void Executor::operator()(const Model::DispatchCommand& command)
         return;
     }
 
+    ID3D12Resource* m_timestampResultBuffers;
+
+    const uint32_t timestampCount = m_device->timestampCount;
+
+    m_device->D3D()->CreateCommittedResource(
+        &CD3DX12_HEAP_PROPERTIES(D3D12_HEAP_TYPE_READBACK),
+        D3D12_HEAP_FLAG_NONE,
+        &CD3DX12_RESOURCE_DESC::Buffer(sizeof(UINT64) * 2 * timestampCount),
```

Comment:
We typically use standard C++ types instead of Windows macros, i.e., `uint64_t` instead of `UINT64`. The exception is for public Windows APIs like directml.h, which have their own special rules.